src/dune_rules/melange/melange_stanzas.ml

```diff
@@ -112,6 +114,22 @@ module Emit = struct
          field_b "allow_overlapping_dependencies"
        and+ emit_stdlib = field "emit_stdlib" bool ~default:true
        and+ modules = Stanza_common.Modules_settings.decode in
+       let () =
+         match alias with
+         | None -> ()
+         | Some a -> (
+           match Alias.Name.equal a implicit_alias with
+           | false -> ()
+           | true ->
+             User_error.raise ~loc:alias_loc
+               [ Pp.textf
+                   "The alias %S is assigned implicitly by Dune to include all \
+                    dependencies generated by all melange.emit stanzas found \
+                    in the project. To use an alias for this particular \
+                    stanza, use a different name."
+                   (Alias.Name.to_string implicit_alias)
```

Comment:
Do we really need this check anymore? I see nothing wrong with writing `(alias melange)` explicitly even if it's useless.

Reply:
Fixed in https://github.com/ocaml/dune/pull/7327/commits/685a9b31d9f92241b6e884cf964cc67f0abe4576.